src/climatevision/generator/entries.py

```diff
@@ -187,3 +187,459 @@ class Entries:
     t_s_ghg_sum_com: float
     w_elec_fec: float
     ags: str
+
+    def assert_is_valid(self):
+        """Checks that the numeric values are not negative"""
+
+        assert (
```

Comment:
Könnte man das auch irgendwie generisch für alle floats und ints machen (z.B. so ähnlich wie in test_entries#create_valid_entries_values). Dort ermittelst du ja auch alle Entries vom Typ float / int.